### PR TITLE
[C++20] converted gsl objects to std

### PR DESCRIPTION
**Pull request by p01arst0rm**
updated gsl objects to use std builtins.

### CODE REVIEW

**Comment by p01arst0rm** [comment]
related: https://github.com/RPCS3/rpcs3/pull/7531

File: rpcs3/Crypto/utils.cpp [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
 #include "Utilities/StrUtil.h"
-#include "Utilities/span.h"
 #include "Utilities/File.h"
 
 #include <memory>
 #include <string>
 #include <string_view>
 
+#include <span>
+//#include "Utilities/span.h"
```
Comment (elad335) [inline rpcs3/Crypto/utils.cpp:19]: No point in comment.

Comment (p01arst0rm) [reply rpcs3/Crypto/utils.cpp:19]: i commented it out instead of removing it just to be safe :)

Comment (elad335) [reply rpcs3/Crypto/utils.cpp:19]: It just confuses who reads it, kind of TMI of unrelated events in code history.

Comment (p01arst0rm) [reply rpcs3/Crypto/utils.cpp:19]: fair .. i also just realised i basically redid the entirety of an existing commit because im an idiot ://

**Comment by p01arst0rm** [comment]
duplicate pull request; closing.

**Pull request closed without merging**